packages/os/storewolf.service

```diff
@@ -1,7 +1,5 @@
 [Unit]
 Description=Datastore creator
-After=migrator.service
```

Comment:
It moved to `RequiredBy` in [migrator.service](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/23/files#diff-aeaa16170f631bfaab1d68685d57d5bbc041cb32330500091950a24dbfe8fdf8).

The problem with listing it here is that - if `migrator.service` is not installed - then `storewolf.service` will fail to start because a required service is missing. So we need to invert the dependencies to make it optional.

Reply:
Ok, that makes sense!